resources/services/security/jit_network_access_policies.go

```diff
@@ -156,6 +156,9 @@ func fetchSecurityJitNetworkAccessPolicyVirtualMachines(ctx context.Context, met
 }
 func resolveSecurityJitNetworkAccessPolicyVirtualMachinesPublicIpAddress(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
 	p := resource.Item.(security.JitNetworkAccessPolicyVirtualMachine)
+	if p.PublicIPAddress == nil {
```

Comment:
Nit: why not use the built in ip resolvers?

Reply:
We could. Are they in the SDK or client?